rocketpy/Environment.py

```diff
@@ -347,6 +348,12 @@ def __init__(
         else:
             self.date = None
 
+        # Save local date
+        self.local_date = None
+
+        # Save local time zone
+        self.time_zone = None
+
```

Comment:
Please add `Environment.local_date` and `Environment.time_zone` to the class docstring near line 85. Example:

```
...
Environment.date : datetime
            Date time of launch in UTC.
Environment.local_date : datetime
            Date time of launch in the local time_zone, defined by Environment.time_zone.
Environment.time_zone : string
            Local time zone specification.

...
```